server/models/Availability.js

```diff
@@ -0,0 +1,11 @@
+const mongoose = require("mongoose");
+
+const availabilitySchema = new mongoose.Schema({
+    day: {type: Number},
+    month: {type: Number},
+    year: {type: Number},
+    from: {type: String},
+    to: {type: String},
+}, { toJSON: { virtuals: true } })
```

Comment:
Not quite the approach I would like to take with availability. Ideally we have something similar to Calendly. 

Example

```
Monday: 9am to 5pm
Tuesday: 9am to 5pm
. . .
```

However, we need to structure this in a way that's easy to query and determine whether a booked time slot is actually available.